ros/src/twist_controller/twist_controller.py

```diff
@@ -78,30 +86,30 @@ def control(self,
                                                                 angular_velocity=angular_velocity,
                                                                 current_velocity=current_velocity)
 
-            corrective_steer = self.steer_pid.step(error=cte, sample_time=sample_time)
+            corrective_steer = self.steer_pid.step(
+                error=cte, sample_time=sample_time)
 
             steer = CORR_STEERING_FACTOR * corrective_steer + PRED_STEERING_FACTOR * predictive_steer
 
             rospy.logdebug('steer = %f, cte = %f, corrective_steer = %f, predictive_steer = %f',
-                          steer, cte, corrective_steer, predictive_steer)
+                           steer, cte, corrective_steer, predictive_steer)
 
             vel_delta = linear_velocity - current_velocity
-            res_accel = self.accel_pid.step(error=vel_delta, sample_time=sample_time)
-            
-            ###### Filter start
-            res_accel = self.avg_filter.filter(res_accel)
-            ###### Filter End
+            res_accel = self.accel_pid.step(
```

Comment:
is it a result of auto formatter?

Reply:
yes.. it's the resultl of autoformater in visual studio code.